src/llmcompressor/transformers/finetune/text_generation.py

```diff
@@ -418,7 +418,10 @@ def main(
 
     # wrap model.save_pretrained
     if is_fsdp_model(model):
-        modify_fsdp_model_save_pretrained(trainer, processor)
+        raise NotImplementedError(
```

Comment:
We should put this in after the oneshot / stage runner refac. Currently this is an ok pipeline.